SpatialGDK/Source/Public/SpatialConstants.h

```diff
@@ -36,4 +36,6 @@ namespace SpatialConstants
 		uint32 WaitTimeExponentialFactor = 1u << (NumAttempts - 1);
 		return FIRST_COMMAND_RETRY_WAIT_SECONDS * WaitTimeExponentialFactor;
 	}
+
+	const FString LOCAL_HOST = TEXT("127.0.0.1");
```

Comment:
Super nit: No underscore, since you usually type `localhost`.

Reply:
Voted here, underscore won <3